event-ticket.html

```diff
@@ -0,0 +1,440 @@
+<!doctype html>
```

Comment:
Can't we get Fullname and Email by asking user to disclose their attribute? that's the thing we want to achieve by DI, right?